modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -15,8 +15,18 @@ Specifically, it protects a cluster from large-scale infrastructure failure, thr
 Each group is created by an appropriately authorized administrator, and specified to contain a subset of the nodes within a Couchbase Cluster.
 Following group-definition and rebalance, the active vBuckets for any defined bucket are located on one group, while the corresponding replicas are located on another group: indexes and _index replicas_, maintained by the Index Service, are distributed across groups in a similar way.
 
-A server group can be automatically _failed over_: thus, if the entire group goes offline, and active vBuckets and indexes are thereby inaccessible, the replica vBuckets and replica indexes that remain available on another group can be automatically promoted to active status.
+A server group can be automatically _failed over_: thus, if the entire group goes offline, and active vBuckets and indexes are thereby inaccessible, then the replica vBuckets and replica indexes that remain available on another group can be automatically promoted to active status.
+
+NOTE: For the vBuckets and replica indexes to be automatically promoted to active, the conditions specified in xref:./automatic-failover.adoc#auto-failover-constraints[Auto-failover Constraints] must apply.
+
+See xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover].
 In 7.1+, this is achieved by setting the maximum count for auto-failover to a value equal to or greater than the number of nodes in the server group.
+
+Note that In 7.1+, _automatic failover_ can fail over more than three nodes concurrently: this has permitted the removal of pre-7.1 interfaces that were specific to triggering _auto-failover for server groups_.
+Consequently, in order to ensure successful auto-failover of a server group, the maximum count for auto-failover must be established by the administrator as a value equal to or greater than the number of nodes in the server group.
+
+Note also that in pre-7.1, interfaces for triggering auto-failover for server groups have been _removed_ from 7.1: therefore, programs that attempt to use the pre-7.1 interfaces with 7.1+ will _fail_. +
```

Comment:
This line -- line 28 -- seems a bit out-of-place here -- best to remove since the previous paragraph already noted the removal of pre-7.1 interfaces.